docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+{
```

Comment:
General tip throughout all notebooks/code/docstrings is:
"predictive probabilities" -> "predicted probabilities"

An even more descriptive names to use occasionally near the start of notebooks include:
- "predicted class probabilities"
- "out-of-sample predicted probabilities"